### ISSUE TITLE
Issue following DashR installation guide.

### ISSUE THREAD

**Issue by michaelbabyn** [plotly/dash-docs#572]
I ran into some trouble trying to install dash according to the instructions on https://dashr.plot.ly/installation. When installing dashR, I got `Error: (converted from warning) packages 'dashCoreComponents', 'dashHtmlComponents', 'dashTable' are not available (for R version 3.4.4)` which prevented dash from being installed until I installed dcc/html and dash-table first.

So I needed to install in the below order instead of what is mentioned in the docs:
```R
library(devtools) # devtools: Tools to Make Developing R Packages Easier

install_github('plotly/dash-html-components') # HTML components
install_github('plotly/dash-core-components') # Supercharged components
install_github('plotly/dash-table')

install_github('plotly/dashR') # The core dash backend
```

**Comment by rpkyle** [comment]
> I ran into some trouble trying to install dash according to the instructions on https://dashr.plot.ly/installation. When installing dashR, I got `Error: (converted from warning) packages 'dashCoreComponents', 'dashHtmlComponents', 'dashTable' are not available (for R version 3.4.4)` which prevented dash from being installed until I installed dcc/html and dash-table first.

@michaelbabyn Thanks for this; I've made a minor change that should help. Can you try removing those packages locally, then reinstalling from the `fix-remotes` branch?

First, try

```
invisible(lapply(list("dash", "dashHtmlComponents", "dashTable", "dashCoreComponents"), remove.packages))
```

...then, try

```
library(devtools) # devtools: Tools to Make Developing R Packages Easier

install_github('plotly/dashR', ref="fix-remotes") # The core dash backend
install_github('plotly/dash-html-components') # HTML components
install_github('plotly/dash-core-components') # Supercharged components
install_github('plotly/dash-table')
```

Let me know if this fixes it!

**Comment by michaelbabyn** [comment]
@rpkyle , this fix works for me.

**Comment by rpkyle** [comment]
Closed via plotly/dashR#102.
